pkg/visitor/fetch_test.go

```diff
@@ -91,7 +97,7 @@ func TestFetch(t *testing.T) {
 		if err != nil {
 			t.Fatalf("Failed to fetch spec contents: %s", err)
 		}
-		if string(spec.Contents) != specContents {
+		if !bytes.Equal(spec.Contents, gzippedSpecContents) {
```

Comment:
This and the following two changes show an inconsistency -- ListSpecs is now (in this PR) returning specs uncompressed. `GetSpecs` was previously downloading specs compressed and returning them without uncompressing. I think it's probably better for `GetSpecs` to also uncompress specs before returning them, but think it might overcomplicate this PR by making that change here.